crates/oxc_linter/src/snapshots/typescript_no_namespace.snap

```diff
@@ -79,217 +79,200 @@ source: crates/oxc_linter/src/tester.rs
   help: Replace the namespace with an ES2015 module or use `declare module`
 
   ⚠ typescript-eslint(no-namespace): ES2015 module syntax is preferred over namespaces.
-   ╭─[no_namespace.tsx:2:7]
- 1 │ 
- 2 │             namespace Foo.Bar {
-   ·             ─────────
- 3 │               namespace Baz.Bas {
+   ╭─[no_namespace.tsx:1:1]
+ 1 │ namespace Foo.Bar {
+   · ─────────
+ 2 │                namespace Baz.Bas {
    ╰────
   help: Replace the namespace with an ES2015 module or use `declare module`
 
   ⚠ typescript-eslint(no-namespace): ES2015 module syntax is preferred over namespaces.
-   ╭─[no_namespace.tsx:3:9]
- 2 │             namespace Foo.Bar {
- 3 │               namespace Baz.Bas {
-   ·               ─────────
- 4 │                 interface X {}
+   ╭─[no_namespace.tsx:2:10]
+ 1 │ namespace Foo.Bar {
+ 2 │                namespace Baz.Bas {
+   ·                ─────────
+ 3 │                  interface X {}
    ╰────
   help: Replace the namespace with an ES2015 module or use `declare module`
 
   ⚠ typescript-eslint(no-namespace): ES2015 module syntax is preferred over namespaces.
-   ╭─[no_namespace.tsx:2:7]
- 1 │ 
- 2 │             namespace A {
-   ·             ─────────
- 3 │               namespace B {
+   ╭─[no_namespace.tsx:1:1]
+ 1 │ namespace A {
+   · ─────────
+ 2 │                namespace B {
    ╰────
   help: Replace the namespace with an ES2015 module or use `declare module`
 
   ⚠ typescript-eslint(no-namespace): ES2015 module syntax is preferred over namespaces.
-   ╭─[no_namespace.tsx:3:9]
- 2 │             namespace A {
- 3 │               namespace B {
-   ·               ─────────
- 4 │                 declare namespace C {}
+   ╭─[no_namespace.tsx:2:10]
+ 1 │ namespace A {
+ 2 │                namespace B {
+   ·                ─────────
+ 3 │                  declare namespace C {}
```

Comment:
Making the test case strings prettier makes the snapshots less pretty. Should I revert the test case formatting changes?

Reply:
The snapshots are mainly used for checking whether the spans are pointing to the correct thing. So no need.